CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### main
+
+* Add `MapboxMapsOptions.get/setWorldview()` and ``MapboxMapsOptions.get/setLanguage()`. Use this to to adjust administrative boundaries/map language based on the map's audience.
```

Comment:
Maybe add a link to the docs so the options are displayed

https://docs.mapbox.com/help/glossary/worldview/ 

https://docs.mapbox.com/help/troubleshooting/change-language/

Reply:
Good point, added the links as well 👍